Source/WebCore/page/Page.h

```diff
@@ -1348,6 +1352,11 @@ class Page : public Supplementable<Page>, public CanMakeWeakPtr<Page> {
 #endif
 
     ContentSecurityPolicyModeForExtension m_contentSecurityPolicyModeForExtension { ContentSecurityPolicyModeForExtension::None };
+    
+#if PLATFORM(COCOA)
+    bool m_joined { false };
```

Comment:
I don't think you need this because you're using call_once instead.

Reply:
Fixed!